### PR TITLE
Remove theia prefix from ClickHouse Operator images

### PR DESCRIPTION
**Pull request by heanlan**
Push clickhouse/clickhouse-server image to antrea registry

Signed-off-by: heanlan <hanlan@vmware.com>

### CODE REVIEW

**Review by elton-furtado — APPROVED** [review]

**Comment by heanlan** [comment]
Also push clickhouse/clickhouse-server image to antrea registry

File: .github/workflows/docker_update_theia.yml [hunk 1]
```diff
@@ -29,18 +32,24 @@ jobs:
...
         uses: akhilerm/tag-push-action@v2.1.0
         with:
           src: docker.io/altinity/metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
           dst: |
-            docker.io/antrea/theia-metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+            docker.io/antrea/metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+      - name: Push antrea/clickhouse-server
+        uses: akhilerm/tag-push-action@v2.1.0
+        with:
+          src: docker.io/clickhouse/clickhouse-server:${{ github.event.inputs.ch-operator-tag }}
```
Comment (yanjunz97) [inline .github/workflows/docker_update_theia.yml:50]:
```suggestion
          src: docker.io/clickhouse/clickhouse-server:${{ github.event.inputs.ch-server-tag }}
```

Comment (dreamtalen) [inline .github/workflows/docker_update_theia.yml:50]: `ch-operator-tag` -> `ch-server-tag`

File: .github/workflows/docker_update_theia.yml [hunk 2]
```diff
@@ -29,18 +32,24 @@ jobs:
...
-      - name: Push antrea/theia-metrics-exporter
+            docker.io/antrea/clickhouse-operator:${{ github.event.inputs.ch-operator-tag }}
+      - name: Push antrea/metrics-exporter
         uses: akhilerm/tag-push-action@v2.1.0
         with:
           src: docker.io/altinity/metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
           dst: |
-            docker.io/antrea/theia-metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+            docker.io/antrea/metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+      - name: Push antrea/clickhouse-server
```
Comment (yanjunz97) [inline .github/workflows/docker_update_theia.yml:47]: Maybe add a comment saying we need this to save official image to decouple the Antrea e2e test dependency on Thiea repo?

Comment (heanlan) [reply .github/workflows/docker_update_theia.yml:51]: done

**Review by dreamtalen — APPROVED** [review]

**Pull request merged**